src/pushsource/_impl/backend/staged/staged_cloud.py

```diff
@@ -0,0 +1,141 @@
+import logging
+import os
+import yaml
+
+from .staged_base import StagedBaseMixin, handles_type
+from ...model import VHDPushItem, AmiPushItem, AmiRelease, BootMode, KojiBuildInfo
+
+LOG = logging.getLogger("pushsource")
+
+
+class StagedCloudMixin(StagedBaseMixin):
+    def __build_ami_push_item(self, resources, src, origin, name):
+        build_resources = resources.get("build")
+        release_resources = resources.get("release", {})
+        build_info = KojiBuildInfo(
+            name=build_resources.get("name"),
+            version=build_resources.get("version"),
+            release=build_resources.get("respin"),
+        )
+        image_kwargs = {
+            "name": name,
+            "src": src,
+            "build_info": build_info,
+            "origin": origin,
+        }
+
+        image_kwargs.update(
+            {
+                "boot_mode": (
+                    BootMode(resources.get("boot_mode"))
+                    if resources.get("boot_mode")
+                    else None
+                )
+            }
+        )
+
+        release_required = ["product", "date", "arch", "respin"]
+        if all(x in release_resources.keys() for x in release_required):
+            release_attrs = [
+                "product",
+                "date",
+                "arch",
+                "respin",
+                "version",
+                "base_product",
+                "base_version",
+                "variant",
+                "type",
+            ]
+            release_kwargs = {}
+            for key in release_attrs:
+                release_kwargs[key] = release_resources.get(key)
+
+            image_kwargs["release"] = AmiRelease(**release_kwargs)
+
+        image_attrs = [
+            "dest",
```

Comment:
Can we please drop "dest" from this list?
Staged source is defined as deriving the "dest" (destination) from the path.
I would rather this be universally consistent and we don't introduce a different way of setting the dest for this content type.

I believe that your client code is expected to ignore the value anyway and set it from elsewhere after the item comes out of pushsource library, so I think this shouldn't affect you, right?